src/erf.jl

```diff
@@ -20,3 +19,41 @@ function erfc(a::Interval{T}) where T
     isempty(a) && return a
     @round( erfc(a.hi), erfc(a.lo) )
 end
+
+function erfinv(a::BigFloat)
+    domain = Interval{BigFloat}(-1, 1)
+    a ∉ domain && return DomainError("$a is not in [-1, 1]")
+    f = x -> erf(x) - a
+    fp = x->2/sqrt(pi_interval(BigFloat)) * exp(-x^2)
+    rts = roots(f, Interval{BigFloat}(-Inf, Inf), Newton(f, fp))
+    @assert length(rts) == 1 && rts[1].status == :unique
+
+    mid(rts[1].interval)
+end
+
+function erfinv(a::Interval{T}) where T
+    domain = Interval{T}(-1, 1)
+    a = a ∩ domain
+
+    isempty(a) && return a
+    Interval{T}(erfinv(a.lo), erfinv(a.hi))
```

Comment:
I don't think this is quite right, since you're not guaranteeing correct rounding by using `mid` in the previous function.

That is why I had suggested:
- `_erfinv(a::BigFloat)` returns an *interval*
- `erfinv(a::BigFloat)` uses `_erfinv` and takes the midpoint
- `erfinv(a::BigFloat, r::RoundingMode) uses `_erfinv` and takes the upper or lower endpoint
- `erfinv(a::Interval)` uses `@round` (I think).